firebase-firestore/src/main/java/com/google/firebase/firestore/bundle/BundleReader.java

```diff
@@ -34,18 +34,19 @@
   /** The capacity for the internal char buffer. */
   protected static final int BUFFER_CAPACITY = 1024;
 
+  private static final Charset UTF8_CHARSET = Charset.forName("UTF-8");
+
   private final BundleSerializer serializer;
-  private final InputStreamReader dataReader;
-  private final Charset charset = Charset.forName("UTF-8");
+  private final InputStream bundleInputStream;
 
   @Nullable BundleMetadata metadata;
-  private CharBuffer buffer;
+  private ByteBuffer buffer;
   long bytesRead;
 
   public BundleReader(BundleSerializer serializer, InputStream data) {
     this.serializer = serializer;
-    dataReader = new InputStreamReader(data, charset);
-    buffer = CharBuffer.allocate(BUFFER_CAPACITY);
+    bundleInputStream = data;
```

Comment:
Can you rename the argument to `bundleInputStream` as well? We could also drop the prefix similar to `serializer`.

Reply:
Done.